ignition/what-is-ignition.md

```diff
@@ -0,0 +1,34 @@
+# What is Ignition? #
+
+Ignition is a new provisioning utility designed specifically for CoreOS. It
```

Comment:
I would suggest teasing apart "what is ignition" from "how does ignition differ to cloudinit". Design for the brave new world where not everyone coming to Ignition will necessarily be familiar with cloudinit (and in particular, its flaws)